packages/create-bison-app/template/graphql/modules/user.ts.ejs

```diff
@@ -53,6 +53,27 @@ export const UserQueries = extendType({
       description: 'Returns the currently logged in user',
       resolve: (_root, _args, ctx) => ctx.user,
     });
+
+    // List Users Query (admin only)
+    t.field('users', {
+      type: list('User'),
+      resolve: async (_root, _args, ctx) => {
+        if (!isAdmin(ctx.user)) throw new ForbiddenError('Unauthorized');
```

Comment:
I don't know enough about nexus to know if this is the best way, but nexus has an `authorize` function you can provide for auth checks. For example:

```js
      authorize: (_root, _args, ctx) => isAdmin(ctx.user),
```

Reply:
You are absolutely right, I did that in one of the other functions and I didn't here :doh: